src/components/clusterWizard/HostsTable.tsx

```diff
@@ -116,17 +131,90 @@ const HostsTable: React.FC<HostsTableProps> = ({ hosts = [], uiState, fetchHosts
     [hostRows, openRows],
   );
 
+  const addAlert = React.useCallback(
+    (alert: Alert) => {
+      if (alert.key) {
```

Comment:
When adding an alert with the same key which already exists, do we want to replace the alert?

Reply:
Good point. With recent implementation, the alert is not duplicated but moved to the top instead.